src/DGmethods/DGmodel.jl

```diff
@@ -1,12 +1,20 @@
-struct DGModel{BL,G,NFND,NFD,GNF}
+struct DGModel{BL,G,NFND,NFD,GNF,AS,DS}
   balancelaw::BL
   grid::G
   numfluxnondiff::NFND
   numfluxdiff::NFD
   gradnumflux::GNF
+  auxstate::AS
+  diffstate::DS
+end
+function DGModel(balancelaw, grid, numfluxnondiff, numfluxdiff, gradnumflux;
+                 auxstate=create_auxstate(balancelaw, grid),
+                 diffstate=create_diffstate(balancelaw, grid))
+  DGModel(balancelaw, grid, numfluxnondiff, numfluxdiff, gradnumflux, auxstate,
+          diffstate)
 end
 
-function (dg::DGModel)(dQdt, Q, param, t; increment=false)
+function (dg::DGModel)(dQdt, Q, _, t; increment=false)
```

Comment:
Looks like an upstream issue (https://github.com/JuliaLang/julia/issues/32727).

```suggestion
function (dg::DGModel)(dQdt, Q, _unused, t; increment=false) # can't use just _ due to #32727
```

Reply:
How about that way if some parameters do come through we know something went wrong?

```suggestion
function (dg::DGModel)(dQdt, Q, ::Nothing, t; increment=false)
```

Reply:
sounds good!